src/core/pretty.ml

```diff
@@ -205,6 +205,10 @@ let pp_command : p_command pp = fun oc cmd ->
       List.iter (out " %a" pp_p_arg) args;
       Option.iter (out " :@ @[<hov>%a@]" pp_p_term) ao;
       out " ≔@ @[<hov>%a@]@]" pp_p_term t
+  | P_inductive(e,s,t,tl)   ->                         (* @Moi *)
```

Comment:
Thanks Amélie to avoid comments like (*@Moi*).

Reply:
It's OK now.